### PR TITLE
Added aws hosting instructons

### PR DESCRIPTION
**Pull request by kopachev**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
 ![alt text](https://raw.githubusercontent.com/RoarData/stanford-faq/master/media/roar-logo.png "ROAR")
```
Comment (lilyS123) [inline FAQ.md]: You think Charles would averse to having the ROAR logo embded here?  Antoine added it but not sure if we want it there.

File: README.md [hunk 2]
```diff
@@ -33,8 +33,8 @@ There are two options for running a Quickstart Jupyter Notebook locally on your
 ## Deployment options
 Once you've built and tested your model, we do not recommend that you run it locally as your bot will disconnect from ROAR contest in the event of your workstation going to sleep or getting shut down.  Hence, it is recommended that you deploy your bot model on a more stable server infrastructure such as the Public Cloud. Some cloud options and detailed deployment steps are provided below.
 
-[heroku](heroku.md) <br/>
-[GCP](gcp.md) <br/> 
+#### [Google Cloud Platform](gcp.md)
+#### [Amazon Web Services](aws.md)
```
Comment (lilyS123) [inline README.md]: I understood Grant wants AWS to also be silent - i.e. we don't link to it from README.

File: aws.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+4. Unzip the SDK and run it
+    - Go to the SSH window and type: `sudo apt install unzip`
+    - To expand the contents type: `unzip Tutorial.zip`
+    - Update Ubuntu package registry: `sudo apt update`
+    - To install Docker in the VM run: `sudo apt install docker.io`
+
+## Run the bot
+1. Now follow the steps from [Build and Run Docker Container](docker.md#docker-install) section.
+   - Note that in the JupyterLab URL shown after the container starts you must substitute the localhost IP with your remote VM IP address, which is visible in "IPv4 Public IP" column of your instance in the AWS Console.
+
```
Comment (lilyS123) [inline aws.md]: Do you want anyone to test drive these steps?  We can ask Ginny to help.

File: heroku.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-Simple instruction to participate to ROAR with `heroku`:
-
-These steps will package and deploy your local setup on the `heroku` platform (by way a Docker image).
+Note that `heroku` is **not** recommended platform for hosting ROAR contest bots 
+at the moment beause the hosted application are restarted daily. This will cause
```
Comment (lilyS123) [inline heroku.md]: because typo

File: heroku.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
-Simple instruction to participate to ROAR with `heroku`:
-
-These steps will package and deploy your local setup on the `heroku` platform (by way a Docker image).
+Note that `heroku` is **not** recommended platform for hosting ROAR contest bots 
+at the moment beause the hosted application are restarted daily. This will cause 
+your bots running in Jupyter notebooks to stop as a result.
```
Comment (lilyS123) [inline heroku.md]: I would add here that they would need to manually start the bot as a workaround

File: FAQ.md (hunk 1 above)
Comment (kopachev) [reply FAQ.md]: @cwRoar

**Pull request merged**